### ISSUE TITLE
install of svn idlutils to get evilmake fails due to no evilmake present

### ISSUE THREAD

**Issue by havok2063** [sdss/sdss_install#44]
My attempt to install the `mangadrp` on a fresh machine failed when attempting to run `evilmake` because it's not available or installed.  I know it's in `idlutils` so I tried installing that product so I could have `evilmake` available.   It also failed with `FileNotFoundError: [Errno 2] No such file or directory: 'evilmake': 'evilmake'`.   

I can't remember if there's another way to install `evilmake` outside of `idlutils`.  Since `evilmake` is a necessary installation tool, maybe we should consider moving it out of `idlutils`, which itself depends on it, and into `sdss_install`.    Or at least have `sdss_install` look for it first and install it if necessary.  Or if `evilmake` can already be installed separately outside of `idlutils`, let's document that somewhere.

**Comment by joelbrownstein** [comment]
history: in svn its possible to checkout just evilmake, and then use that to sdss4install idlutils, and then evilmake is permanent and other products can be sdss4installed.

future: deprecate evilmake since it really only adds a few additional make flags that can be sent to ordinary make (Michael Blanton's suggestion).

**Comment by joelbrownstein** [comment]
suggestion: let's fold this feature (evilmake deprecation) into Utah switch to LUA (since we will need to re sdss_install everything giving us the opportunity to test that new-make does not break individual product installs)

**Comment by joelbrownstein** [comment]
short-term fix: add special instructions on how to manually install idultils without sdss_install as is the solution for SDSS-IV (see https://www.sdss.org/dr15/software/idlutils/)

**Comment by havok2063** [comment]
Deprecating `evilmake` and moving its additional flags into regular `make` sounds reasonable to me.  In the meantime I can install `idlutils` separately.

**Comment by joelbrownstein** [comment]
agreed.  I'll talk to Ben about making a `noevil` branch of sdss_install
